CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 - Use of RIPE RIS firehose stream instead of the websocket clientui8
 - Use of function url_for in flask redirect
 - In hijack view changed the actions functionality
+- Update hasura (1.0.0alpha31 -> 1.0.0alpha41)
```

Comment:
Also describe the rest of the changes in the CHANGELOG

Reply:
done